src/lib/helpers.tsx

```diff
@@ -204,7 +204,7 @@ export function compareDates(
  * @param date
  */
 // TODO(axbarsan): Refactor a part of this into a UI component.
-export function relativeDate(date?: string): ReactElement {
+export function relativeDate(date?: string | number | Date): ReactElement {
```

Comment:
There's no reason to not allow this, as all the functions called in the body also allow these